.github/workflows/docker.yml

```diff
@@ -0,0 +1,24 @@
+name: Build app and push Docker image
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
We should (at least in theory) consider our master branch as stable and deployable, so I build the package from that.